docs/deployment_guide.md

```diff
@@ -68,6 +68,18 @@ python ctms/bin/acoustic_fields.py remove fxaaa_id
 python ctms/bin/acoustic_fields.py add fxa_id
 ```
 
+## Acoustic Newsletters Fields
+
+The mapping between the Basket/CTMS newsletter name and the Acoustic field name is controlled by the `acoustic_newsletter_mapping` table in the database.
+
+In order to add or remove certain mappings, use the `acoustic_newsletters_mapping.py` script:
```

Comment:
Related to my comment above, maybe this could read:

> In order to add or remove certain mappings, use the `acoustic_newsletters_mapping.py` script. See `python ctms/bin/acoustic_newsletters_mapping.py --help` for usage details.